README.md

```diff
@@ -3,15 +3,21 @@ Powerful, flexible, lightweight interface to the BitPay Bitcoin Payment Gateway
 
```

Comment:
There are a lot of changes to the README upcoming from Hamish, I want to hold off on these changes until those changes come through.